### PR TITLE
Joinmarket.md first draft

### PR DESCRIPTION
**Pull request by davterra**
CREATE: coinjoin folder

ADD:
joinmarket.md
terminal image

CHANGE:
samourai to dojo, and give it its own chapter, similar to Electrum.
"Samourai" isnt really part of myNode, and Dojo is separate from Coinjoin topics.

### CODE REVIEW

File: docs/coinjoin/joinmarket.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Overview
+
+JoinMarket is software to create a special kind of bitcoin transaction called a ["CoinJoin"](https://en.bitcoin.it/wiki/CoinJoin). Its aim is to improve the confidentiality and privacy of bitcoin transactions. JoinMarket differs from other popular CoinJoin implementations in a few ways.
+
+The most important differences are:
+
+1) Users can choose to act as a market "taker" and initiate a CoinJoin transaction on demand, or they can act as a market "maker" (and potentially earn fees) by leaving a JoinMarket wallet connected to the network for others to CoinJoin with.
+2) JoinMarket has no central coordinator.
+
+JoinMarket was created by Chris Belcher in 2015, and is currently maintained by Adam Gibson [here](https://github.com/joinmarket-clientserver/joinmarket).
```
Comment (kristapsk) [inline docs/coinjoin/joinmarket.md:12]: URL is wrong, should be https://github.com/JoinMarket-Org/joinmarket-clientserver.

Comment (davterra) [inline docs/coinjoin/joinmarket.md:12]:
```suggestion
JoinMarket was created by Chris Belcher in 2015, and is currently maintained by Adam Gibson [here](https://github.com/Joinmarket-Org/joinmarket-clientserver).
```

Comment (davterra) [reply docs/coinjoin/joinmarket.md]: Thanks! Done

**Comment by abhiShandy** [comment]
merged with JoinMarket branch

**Comment by abhiShandy** [comment]
merged with master

**Pull request closed without merging**